### DISCUSSION TITLE
New Zealand's favorite ice cream flavor measured in terms of amount eaten is what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#323] [Galaxy Trivia] [answered]
**Hint:** 🍦

**Comment by thuongtruong1009** [comment]
According to a survey conducted by Tip Top Ice Cream, one of New Zealand's largest ice cream brands, in 2019, the most popular ice cream flavors in New Zealand were vanilla, hokey pokey, and chocolate. Hokey pokey is a New Zealand specialty flavor that consists of vanilla ice cream mixed with small pieces of honeycomb toffee. It's worth noting that this survey measured popularity based on people's preferences, rather than the amount of ice cream consumed.

**Reply by ParanoidUser** [reply]
Nice hit! What would you get?
![image](https://user-images.githubusercontent.com/5120290/225929727-a14e0fcc-a67c-4bd3-9060-cc4645cddfc6.png)

**Comment by teafoot** [comment, marked as answer]
hokey pokey :+1:
